Bottom: afac38baa887182022de9c013ba5f30a904789cd
Top:    03e126ffdb8b11779aefd5b4792b0a6a701e431a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:44:56 -0500

Refresh of buildbot-ssh-keygen

---
diff --git a/roles/buildbot-master/tasks/main.yml b/roles/buildbot-master/tasks/main.yml
index 3415bab6..ff25c6fb 100644
--- a/roles/buildbot-master/tasks/main.yml
+++ b/roles/buildbot-master/tasks/main.yml
@@ -33,6 +33,12 @@
     env: '{{ buildbot_env }}'
     ports: '{{ buildbot_port_args }}'
 
+- name: Grab ssh public key
+  command: >
+    docker exec '{{ buildbot_container_name }}' cat /home/buildbot/.ssh/id_rsa.pub
+  changed_when: false
+  register: output
+
 - name: Add buildbot_master host
   add_host:
     name: buildbot_master
